module/submissions/helper.js

```diff
@@ -540,10 +540,10 @@ module.exports = class SubmissionsHelper {
                     if(modelName == "submissions") {
                         // Push update submission to kafka for reporting/tracking.
                         this.pushInCompleteSubmissionForReporting(updatedSubmissionDocument._id);
-                    } else {
+                    } else if (modelName == "observationSubmissions") {
```

Comment:
@deepakr123 - Check with Ashwini/Shakthi are we tracking incomplete submission for observations ? If we are then they have to track the same for survey, we can discuss this in the call with Shakthi tomorrow

Reply:
resolved